user/deployment/opsworks.md

```diff
@@ -14,9 +14,12 @@ deploy:
   access-key-id: ACCESS-KEY-ID
   secret-access-key: SECRET-ACCESS-KEY
   app-id: APP-ID
+  region: REGION # (default: us-east-1)
 ```
 {: data-file=".travis.yml"}
 
+Strictly, `region` isn't required but it defaults to `us-east-1` so you'll need it if your stack is elsewhere. You will get an error like "OpsWorks service error: Unable to find app with ID xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" if your stack is not in `us-east-1` and you don't define `region`.
```

Comment:
This is now a little repetitive, how about:

>  region` isn't required but it defaults to `us-east-1`. If your application is located in a different region you will get an error like "OpsWorks service error: Unable to find app with ID xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx".

Or do you think we need to be more clear?

Reply:
Nice. That reads much better. I've incorporated your suggestion :)